src/t8_forest/t8_forest_types.h

```diff
@@ -81,6 +81,8 @@ typedef struct t8_forest
   int                 maxlevel_existing;/**< If >= 0, the maximum occurring refinemnent level of a forest element. */
   int                 do_dup;           /**< Communicator shall be duped. */
   int                 dimension;        /**< Dimension inferred from \b cmesh. */
+  int                 is_incomplete;    /**< Flag to decide whether the forest ist complete or not.
```

Comment:
Please be more specific, what does incomplete/complete mean?

Reply:
Know it should be fine.